src/editor/index.js

```diff
@@ -37,9 +43,11 @@ class PopupSidebar extends Component {
 			trigger_delay,
 			trigger_type,
 			utm_suppression,
+			currentPost,
```

Comment:
Where is `currentPost` used?

Reply:
it's not, I'm so used to ESlint that I don't see unused variables by myself anymore 😬